BlockchainSdk/WalletManagers/Aptos/AptosWalletManager.swift

```diff
@@ -14,40 +14,140 @@ final class AptosWalletManager: BaseManager {
     // MARK: - Private Properties
 
     private let transactionBuilder: AptosTransactionBuilder
+    private let networkService: AptosNetworkService
     
     // MARK: - Init
     
-    init(wallet: Wallet, transactionBuilder: AptosTransactionBuilder) {
+    init(wallet: Wallet, transactionBuilder: AptosTransactionBuilder, networkService: AptosNetworkService) {
         self.transactionBuilder = transactionBuilder
+        self.networkService = networkService
         super.init(wallet: wallet)
     }
     
     // MARK: - Implementation
     
     override func update(completion: @escaping (Result<Void, Error>) -> Void) {
-        completion(.success(()))
+        cancellable = networkService
+            .getAccount(address: wallet.address)
+            .sink(
+                receiveCompletion: { [unowned self] completionSubscription in
```

Comment:
`weak` что бы не падало )
```suggestion
                receiveCompletion: { [weak self] completionSubscription in
```

Reply:
> `weak` что бы не падало )

Ага , уберу ....